plugins/woocommerce-blocks/assets/js/blocks/cart/cart-cross-sells-product-list/cart-cross-sells-product.tsx

```diff
@@ -45,13 +44,23 @@ const CartCrossSellsProduct = ( {
 							saleBadgeAlign={ 'left' }
 							imageSizing={ ImageSizing.SINGLE }
 							isDescendentOfQueryLoop={ false }
+							scale={ 'cover' }
+							aspectRatio={ '1:1' }
 						/>
 						<ProductName
 							align={ '' }
 							headingLevel={ 3 }
 							showProductLink={ true }
 						/>
-						<ProductRating />
+						<ProductRating
+							isDescendentOfQueryLoop={ false }
+							isDescendentOfSingleProductBlock={ false }
+							productId={ product.id }
+							postId={ 0 }
+							shouldDisplayMockedReviewsWhenProductHasNoReviews={
+								false
+							}
```

Comment:
I see these are required fields, but I don't think we should have to pass them @albarin is there any concern with us using this block here?

Reply:
Thanks for your review, @opr. I'll await @albarin's reply before taking next steps.

Reply:
I think it's fine to use it in here.

Reply:
Thanks for confirming this, @albarin.